app.js

```diff
@@ -16,7 +16,7 @@ const env = nunjucks.configure([
 ], {
   autoescape: true,
   express: app,
-  watch: true
+  watch: process.env.WATCH_TEMPLATES
```

Comment:
This change was done to prevent the unit tests from hanging after completion, the watcher was holding on to a file handle so jest refused to exit. Have removed template watching unless explicitly turned on for this reason as we probably don't want it in prod anyway.

Reply:
Yep, good call.